2020/11/06 15:59:13 [debug]  xiaodaimeng/main.go : 30   数据库连接失败
2020/11/06 16:00:14 [debug]  xiaodaimeng/main.go : 30   数据库连接失败
2020/11/06 16:33:37 [info]  xiaodaimeng/main.go : 31   connecting to ws:
2020/11/06 16:33:37 [info]  xiaodaimeng/main.go : 35   dial:websocket: bad handshake
2020/11/06 16:34:38 [info]  xiaodaimeng/main.go : 31   connecting to ws:
2020/11/06 16:34:38 [info]  xiaodaimeng/main.go : 35   dial:websocket: bad handshake
